lib/vtls/nss.c

```diff
@@ -1777,6 +1803,7 @@ static CURLcode nss_setup_connect(struct connectdata *conn, int sockindex)
   CURLcode result;
   bool second_layer = FALSE;
   SSLVersionRange sslver_supported;
+  char *sslver_req_str, *sslver_supp_str;
```

Comment:
These definitions could be moved to body of the `if` statement to limit their scope.

Reply:
OK, fixed that; still in the habit of supporting really old compilers that don't like declarations other than before any code...